backend/src/main/scala/snorri/Book.scala

```diff
@@ -0,0 +1,19 @@
+package snorri
+
+import io.circe.{Encoder, Decoder}
+import io.circe.generic.semiauto._
```

Comment:
```suggestion
import io.circe.generic.semiauto.*
```

Scala 3 syntax is to use `*` for wildcard import.

Reply:
Done